cmd/api/config.go

```diff
@@ -18,46 +18,59 @@ import (
 var configFilename = "config.json"
```

Comment:
TL;DR de-anonymizing config structs, so we can pass more focused parameters to `main` method builders.